### DISCUSSION TITLE
these code thing 😥

### DISCUSSION THREAD

**Discussion by GreenMystic** [ImranR98/Obtainium#1605] [Q&A]
![Screenshot_20240506-085550](https://github.com/ImranR98/Obtainium/assets/136427398/eb3cb9c8-8fa8-4b1a-a518-84ccbb6646f6)
what is this mean?

sc:
https://github.com/KitsunedFox/termux-monet/releases

**Comment by DwainZwerg** [comment]
That you had no internet connection at some point during the download ("connection refused"). ~~Are you using a VPN?~~ Oh, I see in your screenshot: You've used airplane-mode.

**Reply by GreenMystic** [reply]
yes airplane activated but i used wifi as well 
and my wifi isn't have any problem on that time

**Reply by DwainZwerg** [reply]
But if airplane mode is activated, the wifi is also influenced. It is known that Obtainium does not work with airplane-mode (there is also an issue for this (https://github.com/ImranR98/Obtainium/issues/1120 (still open, not exactly the same problem; if this works without airplane-mode we have found a new issue …), https://github.com/ImranR98/Obtainium/issues/55 (already solved), https://github.com/ImranR98/Obtainium/issues/47 (already solved))).

**Reply by GreenMystic** [reply]
I'm baffled but okay 
at least i able to use wifi without need to turn off airplane mode (ADB Hack)
